### PR TITLE
Add Structure101 to Architecture heading

### PR DESCRIPTION
**Pull request by willis7**

### CODE REVIEW

**Comment by ripienaar** [comment]
hmm, looks like this might fit under IDEs or platform something? Mind moving it? Trying to save on the amount of categories

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://coveralls.io/ - Display test coverage reports - free for open source
   * https://scrutinizer-ci.com/ - Continuous inspection platform - free for open source
   * https://codecov.io/ - Code coverage tool (SaaS) free for 1 private project and no restrictions for publics repos
+  
+## IDEs
```
Comment (ripienaar) [inline README.md]: There's already a IDE section :) https://github.com/ripienaar/free-for-dev#ide-and-code-editing

Comment (willis7) [inline README.md]: Sorry - long day!

Comment (ripienaar) [inline README.md]: know that song

**Pull request closed without merging**